lib/screens/home.dart

```diff
@@ -30,14 +28,32 @@ class HomeScreen extends ConsumerStatefulWidget {
 
 class _HomeScreenState extends ConsumerState<HomeScreen> {
   late Future<Query$Page$page> resultFuture;
+  bool isGuestUser = false;
+  bool showTooltip = false;
+  GlobalKey profileIconKey = GlobalKey();
```

Comment:
can be removed now

Reply:
isGuestUser and showToolTip is still used. Removed the other two.